src/index.ts

```diff
@@ -166,3 +166,22 @@ export function isSymbol(value: any): value is symbol {
 export function isUndefined(value: any): value is undefined {
   return typeof value === 'undefined';
 }
+
+/**
+ * Determines if a reference is null or undefined.
+ *
+ * @param value Reference to check
+ */
+export function isNullish(value: any): value is null | undefined {
```

Comment:
@joshgummersall Might `isNil` be a more common name for this?

Reference https://lodash.com/docs/4.17.15#isNil